blueprints/acceptance-test/files/tests/acceptance/__name__-test.js

```diff
@@ -1,4 +1,4 @@
-/* jshint expr:true */
+<% if (jshintExists) { %>/* jshint expr:true */<% } %>
```

Comment:
the `<% } %>` part should probably be at the beginning of the next line to make sure that we don't have the unnecessary line break at the start of the file